### PR TITLE
Test

### PR DESCRIPTION
**Pull request by jimenezfede**

### CODE REVIEW

**Review by wilsuriel03 — APPROVED** [review]

File: client/src/App.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const App = () => {
     axios
       .get("/auth/login/successful")
       .then(({ data: { user } }) => { 
-        console.log("got user",user);
+        // console.log("got user",user);
```
Comment (iamdestinos) [inline client/src/App.tsx:26]: should remove

File: client/src/components/sign-in-form/sign-in-form.component.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 const SignInForm = () => {
   
   const googleSignIn = () => {
-    console.log("Google sign in");
+    // console.log("Google sign in");
```
Comment (iamdestinos) [inline client/src/components/sign-in-form/sign-in-form.component.tsx:6]: should remove

File: client/src/routes/gallery/gallery.component.tsx [hunk 3]
```diff
@@ -12,14 +12,14 @@ const Gallery = () => {
   const [gotImages, setGotImages] = useState(false);
 
   useEffect(() => {
-    axios.get('/images')
+    axios.get('/api/images')
       .then(({ data }) => {
-        console.log('images from database', data);
+        // console.log('images from database', data);
```
Comment (iamdestinos) [inline client/src/routes/gallery/gallery.component.tsx:17]: should remove

File: client/src/routes/gallery/gallery.component.tsx [hunk 4]
```diff
@@ -12,14 +12,14 @@ const Gallery = () => {
...
         setAllImages(data);
       })
       .then(() => {
         setGotImages(true);
-        console.log('images in the state', allImages);
+        // console.log('images in the state', allImages);
```
Comment (iamdestinos) [inline client/src/routes/gallery/gallery.component.tsx:22]: Should remove

File: client/src/routes/movies-shows/movies.component.tsx [hunk 5]
```diff
@@ -12,8 +12,9 @@ const Movies = () => {
 
 
   const getMovies = () => {
-    axios.get('/movies')
+    axios.get('/api/movies')
       .then(({ data }) => {
+        console.log('---->', data);
```
Comment (iamdestinos) [inline client/src/routes/movies-shows/movies.component.tsx:17]: should this be here?

File: server/routes/movies/movies.router.ts [hunk 6]
```diff
@@ -5,7 +5,7 @@ const MovieRouter = express.Router();
 
 // example 
 MovieRouter.get('/', getHorrorMovies);
-MovieRouter.get('/api', getMoviesFromAPI);
+// MovieRouter.get('/movie-api', getMoviesFromAPI);
```
Comment (iamdestinos) [inline server/routes/movies/movies.router.ts:8]: should remove

**Review by iamdestinos — APPROVED** [review]
Lots of useless comments. otherwise Approved

**Pull request merged**